webapp/client/src/stories/MerkzeichenPersonBPage.stories.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
```

Comment:
Could we read the MerkzeichenPersonAPage.stories.js?

Reply:
What do you mean re-add? I don't think I ever deleted it?

Reply:
you are totally right. I overlooked it. Sry 🙈